Frontend/Pages/ManageProducts.razor

```diff
@@ -22,7 +22,8 @@ else
             <tr>
                 <th>Name</th>
                 <th>Description</th>
-                <th>Price</th>
+                <th>Curr.Price</th>
+                <th>Org.Price</th>
```

Comment:
Might be better to name it `Current.Price` instead of `Curr.Price`, for better clarity/readability.